app/src/test/java/nl/tudelft/bbw/APITest.java

```diff
@@ -46,14 +47,20 @@
      */
     @Before
     public final void setUp() throws HashException, BlockAlreadyExistsException {
+
+        API.initializeAPI("Jeff", "iban", RuntimeEnvironment.application);
+
+        list = API.getMyContacts();
+
+
         EdDSAPrivateKey privateKey = ED25519.generatePrivateKey();
-        owner = new User("Jeff", "iban", ED25519.getPublicKey(privateKey));
-        owner.setPrivateKey(privateKey);
-        API.initializeAPI(owner, RuntimeEnvironment.application);
-        list = API.getBlocks(owner);
+        newUser = new Acquaintance("Nick", "iban2", ED25519.getPublicKey(privateKey), new ArrayList<List<Block>>());
 
-        privateKey = ED25519.generatePrivateKey();
-        newUser = new User("Nick", "iban2", ED25519.getPublicKey(privateKey));
+        ArrayList<List<Block>> newUserMultichain = new ArrayList<List<Block>>();
```

Comment:
Change it to: List<List<Block>> newUserMultichain = new ArrayList<List<Block>>();.
Since you also have the same implementation in the implementation